utils/upload_ansible_roles_to_galaxy.py

```diff
@@ -362,10 +445,20 @@ def locally_clone_and_init_repositories(organization, repo_list):
 def main():
     args = parse_args()
 
-    all_role_whitelist = {"rhel7-role-%s" % p for p in PROFILE_WHITELIST}
+    all_product_whitelist = {"%s" % p for p in PRODUCT_WHITELIST}
```

Comment:
`all_product_whitelist` is now a set that contains same elements as `PRODUCT_WHITELIST`. Please remove this.

Reply:
@jan-cerny fixed